### PR TITLE
RAD-39 - Study.orderID name and type differ from core

### PR DESCRIPTION
**Pull request by teleivo**
see https://issues.openmrs.org/browse/RAD-39

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/radiology/Study.java [hunk 1]
```diff
@@ -64,13 +64,13 @@ public Study() {
 		super();
 	}
 	
-	public Study(int id, String uid, int orderID, ScheduledProcedureStepStatus scheduledStatus,
+	public Study(int id, String uid, int orderId, ScheduledProcedureStepStatus scheduledStatus,
```
Comment (sunbiz) [inline api/src/main/java/org/openmrs/module/radiology/Study.java]: Did you want to change this to Integer orderId too? any use of the method will allow int, but may be it'll add consistency to the declaration?

File: api/src/main/java/org/openmrs/module/radiology/db/hibernate/StudyDAOImpl.java [hunk 2]
```diff
@@ -42,9 +42,8 @@ public Study getStudy(Integer id) {
 	/* (non-Javadoc)
 	 * @see org.openmrs.module.radiology.db.StudyDAO#getStudyByOrderId(java.lang.Integer)
 	 */
-	@Override
-	public Study getStudyByOrderId(Integer id) {
-		String query = "from Study s where s.orderID = '" + id + "'";
+	public Study getStudyByOrderId(Integer orderId) {
```
Comment (sunbiz) [inline api/src/main/java/org/openmrs/module/radiology/db/hibernate/StudyDAOImpl.java]: Can you also add the @Override annotation again?

File: api/src/main/java/org/openmrs/module/radiology/Study.java (hunk 1 above)
Comment (teleivo) [inline api/src/main/java/org/openmrs/module/radiology/Study.java]: yes, you are right :+1: will update it right now

File: api/src/main/java/org/openmrs/module/radiology/db/hibernate/StudyDAOImpl.java (hunk 2 above)
Comment (teleivo) [inline api/src/main/java/org/openmrs/module/radiology/db/hibernate/StudyDAOImpl.java]: will do as well :+1:

**Comment by teleivo** [comment]
@sunbiz fixed what you commented :+1:

**Comment by sunbiz** [comment]
@teleivo , you are awesome and fast!!

**Pull request merged**